contracts/liquidity_hub/pool-network/stableswap_3pool/src/helpers.rs

```diff
@@ -126,83 +124,6 @@ pub struct OfferAmountComputation {
     pub protocol_fee_amount: Uint128,
     pub burn_fee_amount: Uint128,
 }
-
-/// If `belief_price` and `max_spread` both are given,
-/// we compute new spread else we just use terraswap
-/// spread to check `max_spread`
-pub fn assert_max_spread(
```

Comment:
moved to packages/white-whale/src/pool_network/swap.rs since this code is shared with the pair contract as well.